docs/management/release.rst

```diff
@@ -10,11 +10,17 @@ person doing the release must ensure that the version of the new catalog matches
 following all steps below. Steps 1 and 2 below should be done in a PR and merged before commencing step 3. If the release does 
 not include an update to the catalog on Gadi, skip the first two steps below:
 
-#. [IF UPDATING THE CATALOG] Create a new version of the catalog on Gadi (this will take about 1 hour)::
-
+#. [IF UPDATING THE CATALOG] Create a new version of the catalog on Gadi (this will take about 1 hour)
+   ::
       $ export RELEASE=vX.X.X
       $ cd bin
       $ qsub -v version=${RELEASE} build_all.sh
+   .. note:: 
+      If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To download and cache the schema, first import the `access_nri_intake.source` and `access_nri_intake.catalog` sub-packages from a Gadi node with network access (e.g. a login or ARE node). I.e., using the release version of `access_nri_intake`
```

Comment:
```suggestion
      If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To download and cache the schema, first import the :code:`access_nri_intake.source` and :code:`access_nri_intake.catalog` sub-packages from a Gadi node with network access (e.g. a login or ARE node). I.e., using the release version of :code:`access_nri_intake`
```

Reply:
Oh I just re-read this. Is the 'using the release version of access_nri_intake' correct? In your comment above I thought it need to be the development version ?

Reply:
Release version is right